packages/cardano-services/src/run.ts

```diff
@@ -52,6 +53,10 @@ const envSpecs = {
   SERVICE_DISCOVERY_BACKOFF_FACTOR: envalid.num({ default: SERVICE_DISCOVERY_BACKOFF_FACTOR_DEFAULT }),
   SERVICE_DISCOVERY_TIMEOUT: envalid.num({ default: SERVICE_DISCOVERY_TIMEOUT_DEFAULT }),
   SERVICE_NAMES: envalid.str({ example: Object.values(ServiceNames).toString() }),
+  TOKEN_METADATA_CACHE_TTL: envalid.makeValidator(cacheTtlValidator)(
+    envalid.num({ default: DEFAULT_TOKEN_METADATA_CACHE_TTL })
+  ),
+  TOKEN_METADATA_SERVER_URI: envalid.url({ default: DEFAULT_TOKEN_METADATA_SERVER_URI }),
```

Comment:
```suggestion
  TOKEN_METADATA_SERVER_URL: envalid.url({ default: DEFAULT_TOKEN_METADATA_SERVER_URL }),
```

I've just noticed the inconsistency, which we should correct. Since it's already in _master_, you can do a separate `refactor` commit to align it with our other variables that use "URL". Sorry for not picking this up in the prior PRs

Reply:
4a2bf47fe refactors exactly the same variable (plus some more). 0bbc2e63c (fixup for the same commit) should solve this